templates/archives/stories/digital_humanities-teaser.jinja2

```diff
@@ -0,0 +1,11 @@
+{% extends 'archives/story_card.jinja2' %}
+
+{% block story_title %}
+Digital Humanities Through the Ages (or, Just One Age)
```

Comment:
Maybe this can be re-framed. These scholars were using computers but a very early version of computers, and people had different levels of access as you mention in the story. It may be worth discussing the Humanities Computing field which dates back to the 1930s (see Wikipedia's summary: [https://en.wikipedia.org/wiki/Digital_humanities#History](https://en.wikipedia.org/wiki/Digital_humanities#History) or these articles: [https://modernismmodernity.org/forums/posts/search-and-replace](https://modernismmodernity.org/forums/posts/search-and-replace), [http://digitalhumanities.org/dhq/vol/3/3/000065/000065.html](http://digitalhumanities.org/dhq/vol/3/3/000065/000065.html))

Reply:
By the way, I came across this flyer for a programming course related to linguistic research that might be interesting for you.
http://127.0.0.1:8000/archives/doc/679
[1_28_cc_bulletins_20.pdf](https://github.com/dhmit/computation_hist/files/3099267/1_28_cc_bulletins_20.pdf)

Reply:
That is cool! I have not seen any other documents like that

Reply:
Can you maybe look at again to see if it's more to your liking? Thanks so much for looking this over everyone!